layouts/services/single.html

```diff
@@ -16,26 +16,41 @@ <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl">{{ .approach.title }}</h3>
         </div>
         <p class="mt-4 text-base sm:mt-5 lg:mt-7 lg:text-xl">{{ .approach.description | markdownify }}</p>
         {{ end }}
+
+        {{ if .action.enable }}
+        {{ with .action }}
+        <section class="my-9 md:flex md:justify-between items-center pl-4 py-6 rounded-lg bg-[#E3E3E1]">
+            <p class="pb-6 md:pb-0 font-bold md:text-xl">{{ .description }}</p>
+            <div>
+                <a href="{{ .schedule_link }}" target="_blank" class="bg-[#192E5B] hover:bg-[#1D65A6] p-4 mx-2 rounded-lg font-bold text-[#FCFCFC] md:text-base">{{ .button_text }}</a>
```

Comment:
The `bg` colors here were updated to be consistent with other "buttons" displayed on the site.